frontend/src/components/CategoryItem/CategoryItem.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css, Theme } from '@emotion/react';
+
+const categoryItem = ({ colors, flex }: Theme) => css`
+  ${flex.row}
+
+  justify-content:space-around;
```

Comment:
```suggestion
  justify-content: space-around;
```

앗 여기는 prettier 적용이 안되는 걸까요?!
이 PR은 여기서 머지하고 이건 다음에 반영해도 좋을 것 같아요~!